functions/Slack/main.py

```diff
@@ -27,8 +29,10 @@ def handle(event, context):
     slack_channel = os.environ['COFFEE_BUTTON_SLACK_CHANNEL']
 
     if event['clickType'] == str(ButtonClickType.Single):
+        now = datetime.datetime.now(tz=pytz.timezone('US/Eastern'))
+        ready_time = now + datetime.timedelta(minutes=5)
         requests.post(slack_webhook_url, json={
-            'text': 'Coffee is brewing and will be ready in about 5 minutes!',
+            'text': 'Coffee is brewing and will be ready at {}!'.format(ready_time.strftime('%-I:%M%p')),  # NOQA
```

Comment:
Hmm.  "The full set of format codes supported varies across platforms, because Python calls the platform C library’s strftime() function, and platform variations are common." (https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior)

Are you on a Mac?  I guess even if Lambda supports `%-I` it's probably worth doing the more-compatible thing.